mountpoint-s3-crt/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## v0.5.2 (January 12, 2024)
+
 * Update to latest CRT dependencies
 * Return type of `common::Uri::host_port` changed from `u16` to `u32`
```

Comment:
Should this be a minor version bump instead of patch version?

Reply:
It could be, but rules for unstable (0.x.y) releases are not so strict (I think).

Reply:
This should be a minor version bump for sure (it's API-incompatible).